pkg/assets/assets.go

```diff
@@ -67,13 +85,37 @@ func MustCreateAssetFromTemplate(name string, template []byte, config interface{
 }
 
 func assetFromTemplate(name string, tb []byte, data interface{}) (Asset, error) {
```

Comment:
nit: can we just return *Asset ? then in case of error we don't have to return empty struct...

Reply:
that's your code ;-) But yes, we can :)

Reply:
bootkube code, but yeah, I was lazy :)

Reply:
done